salt/cloud/clouds/saltify.py

```diff
@@ -74,3 +79,46 @@ def get_configured_provider():
         __active_provider_name__ or 'saltify',
         ()
     )
+
+
+def _verify(vm_):
+    '''
+    Verify credentials for an exsiting system
+    '''
+    log.info('Testing logon credentials for {0}'.format(vm_['name']))
+
+    win_installer = config.get_cloud_config_value(
+        'win_installer', vm_, __opts__
+    )
+
+    if win_installer:
```

Comment:
Is this the canonical way to get the platform of the provisioned system? I'm not sure so I'll also ask @gtmanfred @rallytime or @techhat